Cargo.toml

```diff
@@ -21,13 +21,13 @@ void = "1.0.2"
 cc = "1"
 
 [dev-dependencies]
-bytes = "0.4.8"
+bytes = "0.4"
 lazy_static = "1"
 rand = "0.5"
 tempfile = "3"
 
 [target.'cfg(target_os = "freebsd")'.dev-dependencies]
-sysctl = "0.1"
+sysctl = "0.2"
```

Comment:
Why did you update sysctl and bytes?  What was wrong with the old versions?

Reply:
This PR is open so long I don't remember anymore.

On the other side what is wrong with the new versions?

Reply:
It's unnecessarily restrictive.  It might conflict with a crate that depends on nix and also has a restrictive requirement like `sysctl = "0.1.0"` or `sysctl < "0.2"`.

Reply:
Isn't the same true the other way around? If I use `bytes = 0.2` I would still get a copy of `0.1`  because of nix?

Reply:
Nope.  The dependency resolution is a little unintuitive.  Specifying "0.1" really means anything in the half-open range [0.1 - 1.0).  So version 0.2.0 would match.  If you want exactly 0.1.0, then you need to say "=0.1.0".  If you want anything in the range [0.1 - 2.0), you would specify "0.1.0".

Reply:
Are you sure this is true for `0.x` too? According to the [documentation](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements):

> While SemVer says there is no compatibility before 1.0.0, Cargo considers 0.x.y to be compatible with 0.x.z, where y ≥ z and x > 0.

But the example mentions:

> `^0.2 := >= 0.2.0 < 0.3.0`

Which seems kind of contradicting?

Reply:
I removed the update commit. Expect for the `rand` crate.
It's not needed for this.